ios/FluentUI/Extensions/EnvironmentValues+Extensions.swift

```diff
@@ -24,17 +24,19 @@ private extension UIEdgeInsets {
     }
 }
 
-private extension UIApplication {
-    var keyWindow: UIWindow? {
-        connectedScenes
-            .compactMap {
-                $0 as? UIWindowScene
-            }
-            .flatMap {
-                $0.windows
-            }
-            .first {
-                $0.isKeyWindow
-            }
+struct ViewControllerHolder {
+    weak var value: UIViewController?
+}
+
+struct ViewControllerKey: EnvironmentKey {
+    static var defaultValue: ViewControllerHolder {
+        return ViewControllerHolder(value: UIApplication.shared.keyWindow?.rootViewController)
+    }
+}
+
+extension EnvironmentValues {
+    var viewController: UIViewController? {
```

Comment:
nit: how about naming this as something like "ParentViewController" or "HostingViewController"?